.evergreen/install-dependencies.sh

```diff
@@ -90,20 +90,18 @@ else
   mv "${NODE_ARTIFACTS_PATH}/${node_directory}" "${NODE_ARTIFACTS_PATH}/nodejs"
 fi
 
-export PATH="$NODE_ARTIFACTS_PATH/npm_global/bin:$NODE_ARTIFACTS_PATH/nodejs/bin:$PATH"
-hash -r
-
-# Set npm -g prefix to our local artifacts directory
-cat <<EOT > .npmrc
-prefix=$NODE_ARTIFACTS_PATH/npm_global
-EOT
+if [[ -z "${npm_global_prefix}" ]]; then
+  echo "npm_prefix_config not set"
+  exit 1
+fi
```

Comment:
I'm going to port this to the driver.  

I realized that without checking that the correct env variable is set, we could easily silently go back to installing `npm` globally.

Reply:
We also rely on NODE_ARTIFACTS_PATH in this script, should we error check both below the sourcing of init-node-env?

Reply:
Done.